examples/web-wagmi/src/components/feed/Feed.tsx

```diff
@@ -44,7 +25,7 @@ export function Feed() {
       {infiniteScroll.data
         .filter((i) => isPostPublication(i.root))
         .map((item, i) => (
-          <Publication key={`${item.root.id}-${i}`} feedItem={item} />
+          <LinkedPublicationCard key={`${item.root.id}-${i}`} publication={item.root} />
```

Comment:
Let's use a non-link card here. @reecejohnson is reworking these examples to be more of a showcase of all the hooks rather than browseable app. I know I advocated for making these links, but it turns out it does not worth the effort and introduces issues with nested `<a>` which we don't need to deal with.